#the first and second row of inmoov_moveable_joints_names array
inmoov_moveable_joints_counterpart: [12,12,59,14,14,15,
14,18,22,25,29,33,15,
#/the third and the fourth row of inmoov_moveable_joints_names_array
37,37,38,37,41,45,48,52,56,38]

#the first and second row of inmoov_moveable_joints_directions array
inmoov_moveable_joints_direct: [0,1,2, 1,0,3,
2,1,2,2,2,2,0,
#/the third and the fourth row of inmoov_moveable_joints_directions_array
1,0,3,2,1,2,2,2,2,0]
#direct 3 is for bicep
#direct 10 is for waist
#15 && 38 stands for forearms, use waists(16 && 39) to compensate the rotation between forearm and arm. 
#sudden changes between -180 and 180 degrees will occur for 15 && 38. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!


link_useful_direction: [6,6,6,6,6,6
6,6,6,6,6,6,6,
6,6,6,6,6,6,6,6,6,6]